### PR TITLE
ADD: bioconductor-curatedovariandata recipe

### PR DESCRIPTION
**Pull request by gwaygenomics**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by gwaygenomics** [comment]
@bgruening @johanneskoester looks like this is good to merge!

File: recipes/bioconductor-curatedovariandata/build.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+
+#!/bin/bash
+
+# R refuses to build packages that mark themselves as
```
Comment (bgruening) [inline recipes/bioconductor-curatedovariandata/build.sh]: Can you please remove the comments and the leading empty line.

**Comment by gwaygenomics** [comment]
Thanks @bgruening, I made the changes

**Comment by bgruening** [comment]
@gwaygenomics awesome! Thanks a bunch!

**Pull request merged**